### PR TITLE
Use old, unused, code on advanced setting form (Profiles)

### PR DESCRIPTION
**Pull request by braders**
Overview
----------------------------------------
Use old, unused, code on advanced setting form (Profiles)

Before
----------------------------------------
When editing the settings for a profile, dynamic property deprecation notices are thrown (PHP 8.2+):

Screenshot 2024-08-31 at 17 24 21

After
----------------------------------------
The code which is using these dynamic properties is removed, therefore the deprecation notices they no longer occur.

Technical Details
----------------------------------------
An alternative, perhaps safer, option would have been to keep the code in place, and instead define the properties. However, I'm almost certain this code is not used:

1. No commit have changed code referencing `_cmsId` since the import from SVN 12 years ago (as per `git log -S _cmsId`)
2. The code doesn't really make sense - it feels like it was probably initially designed for when the form is submitted, rather than a configuration screen
3. `$form` in this case refers to `CRM_UF_Form_Group`, which appears to never have `_cId` set (I've looked though all the parent classes, traits etc and see no reference to `_cId`)

So in conclusion, I think this code removal is safe.

### CODE REVIEW

**Comment by demeritcowboy** [comment]
For the record, it looks like it was on the form originally: https://github.com/civicrm/civicrm-svn/commit/b2445929f658c23d28ad4fdf8c4bd0d1cfa96ba9

I like the function down at the bottom of that commit. We should bring that back.
